inc/functions/options.php

```diff
@@ -558,28 +558,32 @@ function get_rocket_exclude_async_css() {
  * @return bool true if everything is ok, false otherwise
  */
 function rocket_valid_key() {
-    if ( ! $rocket_secret_key = get_rocket_option( 'secret_key' ) ) {
-        return false;
-    }
+	if ( ! $rocket_secret_key = get_rocket_option( 'secret_key' ) ) {
+		return false;
+	}
 
-	return 8 == strlen( get_rocket_option( 'consumer_key' ) ) && hash_equals( $rocket_secret_key, hash( 'crc32', get_rocket_option( 'consumer_email' ) ) );
+	return 8 === strlen( get_rocket_option( 'consumer_key' ) ) && hash_equals( $rocket_secret_key, hash( 'crc32', get_rocket_option( 'consumer_email' ) ) );
 }
 
 /**
  * Determine if the key is valid
  *
  * @since 2.2 The function do the live check and update the option
+ *
+ * @param string $type Transient type.
+ * @param string $data Data.
+ * @return array WP Rocket options array
  */
 function rocket_check_key( $type = 'transient_1', $data = null ) {
```

Comment:
This function has been updated in the 2.9.7. The content of this one is the old content.